x-pack/test/functional/services/ml/alerting.ts

```diff
@@ -100,5 +101,47 @@ export function MachineLearningAlertingProvider(
         await testSubjects.existOrFail(`mlAnomalyAlertPreviewCallout`);
       });
     },
+
+    async assertLookbackInterval(expectedValue: string) {
+      const actualValue = await testSubjects.getAttribute(
```

Comment:
The fields of the advanced section are present in the DOM even though the section is not expanded, so this works. I would still suggest to `ensureAdvancedSectionOpen` first to make it more stable for potential future changes of that behavior.